contracts/contracts/strategies/balancer/BaseAuraStrategy.sol

```diff
@@ -0,0 +1,140 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+/**
+ * @title OETH Base Balancer Abstract Strategy
+ * @author Origin Protocol Inc
+ */
+
+import { BaseBalancerStrategy } from "./BaseBalancerStrategy.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { IRateProvider } from "../../interfaces/balancer/IRateProvider.sol";
+import { IERC20 } from "../../utils/InitializableAbstractStrategy.sol";
+import { IERC4626 } from "../../../lib/openzeppelin/interfaces/IERC4626.sol";
+import { StableMath } from "../../utils/StableMath.sol";
+import { IRewardStaking } from "../IRewardStaking.sol";
+
+abstract contract BaseAuraStrategy is BaseBalancerStrategy {
+    using SafeERC20 for IERC20;
+    using StableMath for uint256;
+
+    /// @notice Address of the Aura rewards pool
+    address public immutable auraRewardPoolAddress;
+
+    // renamed from __reserved to not shadow BaseBalancerStrategy.__reserved,
+    int256[50] private __reserved_2;
```

Comment:
Open Zeppelin likes the storage slots used + the gap number to equal a nice even number. This is different than starting with a nice even number.

From past audits:

> the gap typically expands the storage size to a round number (like 50 or 100) so all versions of the contract have a predictable size, and the storage layout of descendant contracts starts at a predictable slot number.

Reply:
Rather than using a number, perhaps we should use something like "__reserved_BaseAuraStrategy"?

Reply:
wow, I'd be tempted to use 13 just to trigger a response :-)

Reply:
@naddison36 haha yes. 

@DanielVF agreed, changed the slots in `BaseBalancerStrategy` to 48 for a total of 50. Leaving the BaseAuraStrategy at 50 since it doesn't have any non inherited storage variables.